### PR TITLE
The Next Web Index page Clone

### PR DESCRIPTION
**Pull request by rukundoeric**
### What does this do? 
*  Build with Responsive Design
### Description of Task to be completed? 
* A structural clone of  The Next Web
### How should this be manually tested? 
* Clone this repo  https://github.com/rukundoeric/Building-with-Responsive-Design
* Navigate to project directory.
* If you have a live server in you vs-code, just run it.
* Otherwise just open the index page with your browser.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+    font-family: "Rajdhani", sans-serif;
+    text-align: center; }
+  footer .info label {
+    text-align: center; }
+  footer .info .link-k {
+    color: #fdfdfd;
+    margin-left: 10px;
+    text-transform: uppercase; }
+
+/*# sourceMappingURL=style.css.map */
```
Comment (ghost) [inline css/style.css:865]: Remove the comment on line 845.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Great work on your project. I enjoyed reviewing it.**

**Please make the changes below.**

1). **You have used the wrong breaking points for the mobile and medium screens.**

- **Large screen: min-width - 1024px.**

- **Medium screen: min-with - 768px.**

- **Mobile screen: max-width - 767px**

**When you check the original page and your page, using the above breaking points, you will notice that most parts of your page do not adjust with respect to the change of widths.**

**Here is an example of the latest news section at 768px width.**

original | clone
---------| -------
latestArticle_original | latestArticle_clone

- **Another example at 767px width, which is the max-width of the mobile screen.**

original | clone
---------| -------
hardfork_original | hardfork_clone

2). **Give your pull request a meaningful title.**

### Happy coding 😃.

**PS: You can reach me on slack (Rose Wanjohi)**

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+> A structural clone of the The Next Web home page.
+> ![screenshot](./img/screen_short.JPG)
+
+## Built With
+
+- HTML
+- CSS3
+- Fontawesome
+
```
Comment (javitocor) [inline README.md]: you can add SASS here, do not forget to add the techs used in your project

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,855 @@
...
+  margin: 0 auto;
+  box-sizing: border-box; }
+  @media (max-width: 420px) {
+    .container {
+      width: 100%; } }
+  @media (max-width: 576px) {
+    .container {
+      width: 100%;
+      padding: 0 !important; } }
+  @media (max-width: 812px) {
```
Comment (javitocor) [inline css/style.css]: Do not use the `!important`, it is not a good practice, this word should only be used in very special cases.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,855 @@
...
+  .orthers .more {
+    position: relative; }
+    .orthers .more .more-l {
+      position: relative;
+      margin-left: 20px;
+      width: 94%; }
+      .orthers .more .more-l .li-more {
+        color: #f42;
+        list-style: disc !important;
+        list-style-position: inherit !important;
```
Comment (javitocor) [inline css/style.css]: Same here!

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,855 @@
...
+            top: 9px; }
+            @media (max-width: 768px) {
+              .lastest-deals .deals-list .item .info .old-price .line {
+                left: 12px; } }
+            @media (max-width: 768px) {
+              .lastest-deals .deals-list .item .info .old-price .line {
+                width: 20px; } }
+        .lastest-deals .deals-list .item .info .current-price {
+          color: #272727 !important;
+          font-size: 18px; }
```
Comment (javitocor) [inline css/style.css]: same here

File: scss/_common.scss [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+  box-sizing: border-box;
+
+  @include breakpoint($xs_screen) {
+    width: 100%;
+  }
+
+  @include breakpoint($sm_screen) {
+    width: 100%;
+    padding: 0 !important;
+  }
```
Comment (javitocor) [inline scss/_common.scss]: same here

File: scss/_latest_deals.scss [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+            @include breakpoint($tab_screen) {
+              width: 20px;
+            }
+          }
+        }
+
+        .current-price {
+          color: $gray_color_1 !important;
+          font-size: 18px;
```
Comment (javitocor) [inline scss/_latest_deals.scss]: same here

File: scss/_others_section.scss [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+
+    .more-l {
+      position: relative;
+      margin-left: 20px;
+      width: 94%;
+
+      .li-more {
+        color: $accent_color;
+        list-style: disc !important;
+        list-style-position: inherit !important;
```
Comment (javitocor) [inline scss/_others_section.scss]: same here

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team! Good job with the changes implemented 👍 
Your project is looking much better 💯 

But we still have to implement some more changes to approving this proejct

Kindly, check my comments below 👇  implement the required changes and resubmit your project for a new code review ⏩ 

## Changes

- The breakpoint need to be fixed, there is one breakpoint at 1024px and one more at 768px. Your first breakpoint is at 1200px, please use the breakpoints from the original site. This is a project to practice with responsiveness, so it is important that every breakpoint corresponds with the original site to clone.
- Fix the horizontal margins in all the sections, at this moment you have too much space in the right and left of the screen. This is applicable for all screen sizes. Kindly, modify the margins to make it look like the original. See the screenshots below:
Original:
Capture

Your project:
Capture1

- After the first breakpoint at 1024 the 'latest fundings' section must have only 3 items, now it stills has four. This is also applicable for the next breakpoint. In that section, you must have only 4 items in screens bigger than 1024px. Please remove the extra item.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Great job implementing the requested changes. However, there are still a few needed.**

1). **The min-width for large screen size should be 1024px and not 1025px.**
2). **The min-width for medium screen should be 768px and not 769px.**

3). **As explained by the previous tse, the latest funding rounds section should have three cards at 1024px and below. Also, the small titles should be placed below the large title.**

original | clone
---------| -------
latest_original | latest_clone

- **Please note that the small titles also change the design at 768px width.**

### Happy coding 😃.

**Review by ghost — APPROVED** [review]
# Status: Approved

**Congratulations team and good luck on your next project 😃.**

**Pull request merged**